shadows/framework/src/main/java/org/robolectric/shadows/ShadowApplicationPackageManager.java

```diff
@@ -2199,12 +2248,12 @@ protected String[] setPackagesSuspended(
 
   @Implementation(minSdk = Q)
   @HiddenApi
-  protected /* String[] */ Object setPackagesSuspended(
-      /* String[] */ Object packageNames,
-      /* boolean */ Object suspended,
-      /* PersistableBundle */ Object appExtras,
-      /* PersistableBundle */ Object launcherExtras,
-      /* SuspendDialogInfo */ Object dialogInfo) {
+  protected @ClassName("java.lang.String[]") Object setPackagesSuspended(
```

Comment:
Why not just use `String[]` for the return type?

Reply:
done